src/platform/telink/BLEManagerImpl.cpp

```diff
@@ -467,13 +467,6 @@ void BLEManagerImpl::_InitGatt(void)
     bls_att_setAttributeTable((u8 *) gattTable);
 }
 
-CHIP_ERROR _Shutdown(void)
```

Comment:
This isn't even a class member, it's just totally dead.

Reply:
I don't really know either, I'm just deleting it because it's dead code. I don't think it does any harm, and trying to actually fix it seems futile because.. dead code.

Reply:
Oh, good point about this not being a class member.  OK, then.